lib/DEPENDENCIES.md

```diff
@@ -9,5 +9,6 @@ layerlens.dart-->src;
 ```
 
 Inversions in this folder: 0
+
 Inversions including subfolders: 0.
```

Comment:
You could also format this as:

```
### Inversions

* In this folder: **0**
* Including subfolders: **0**
```

(Or similar)

Reply:
Cool!